harmonica/forward/prism_layer.py

```diff
@@ -22,6 +22,7 @@ def prism_layer(
     surface,
     reference,
     properties=None,
+    progressbar=False,
```

Comment:
This argument is unused and we don't actually need it. Users that want a progressbar while computing the gravity fields of a prism layer would need to pass it to the `gravity` method of the Dataset Accessor.

```suggestion
```

Reply:
Okay I thought that was necessary, but it appears to work fine without.